diff --git a/node_modules/@grpc/grpc-js/build/src/client-interceptors.js b/node_modules/@grpc/grpc-js/build/src/client-interceptors.js
index b39b9c4..cd989d1 100755
--- a/node_modules/@grpc/grpc-js/build/src/client-interceptors.js
+++ b/node_modules/@grpc/grpc-js/build/src/client-interceptors.js
@@ -287,6 +287,11 @@ class BaseInterceptingCall {
                     (_a = interceptingListener === null || interceptingListener === void 0 ? void 0 : interceptingListener.onReceiveMessage) === null || _a === void 0 ? void 0 : _a.call(interceptingListener, deserialized);
                 }
                 catch (e) {
+                    console.log('CLIENT INCERCEPTOR ERROR FOR METHOD: ' + this.methodDefinition.responseDeserialize.toString());
+                    console.log('CLIENT INTERCEPTOR ERROR: ' + e.message);
+                    console.log('CLIENT INTERCEPTOR STACK: ' + e.stack);
+                    console.log('CLIENT INTERCEPTOR BUFFER JSON: ' + JSON.stringify(message));
+                    console.log('CLIENT INTERCEPTOR BUFFER STRING: ' + message.toString('utf8'));
                     readError = {
                         code: constants_1.Status.INTERNAL,
                         details: 'Failed to parse server response',
